docs/installation-and-operation/serialization.md

```diff
@@ -4,53 +4,229 @@ redirect_from:
   - /docs/latest/enterprise-guide/serialization
 ---
 
-## Serialization
+# Serialization
 
 {% include plans-blockquote.html feature="Serialization" self-hosted-only="true" %}
 
-Once you really get rolling with Metabase it's often the case that you'll have more than one instance of it spun up. You might have a couple of testing or development instances and a few production ones, or maybe you have a separate instance per office or region.
+Once you really get rolling with Metabase, it's often the case that you'll have more than one Metabase instance spun up. You might have a couple of testing or development instances and a few production ones, or maybe you have a separate Metabase per office or region.
 
-To help you out in situations like this, Metabase has a serialization feature which lets you create a snapshot, called a dump, of the contents of a Metabase instance that can then be loaded into another instance. This lets you do things like create a set of dashboards and charts in one Metabase instance and then easily copy those items to a number of other Metabase instances. You could also use this feature to enable a staging-to-production workflow for important dashboards or reports by dumping from a staging instance of Metabase and then loading that dump into your production instance(s). You can even put the dump files into version control and audit changes to them, as the YAML files contained within the dump are pretty readable.
+To help you out in situations like this, Metabase has a serialization feature which lets you create an _export_ of the contents of a Metabase that can then be _imported_ into one or more Metabases.
 
-If you're looking to do a one-time migration from H2 to MySQL/Postgres, then use the [migration guide instead](./migrating-from-h2.md).
+## Serialization use cases
 
-## What gets dumped and loaded
+- **Staging environments**. Enable a staging-to-production workflow for important dashboards by exporting from a staging instance of Metabase and then importing them into your production instance(s).
+- **Version control**. Check the exported files into version control and audit changes to them, as the YAML files contained within the export are pretty readable.
 
-Only some artifacts are included in dumps and loads:
+## What gets exported
 
-- Collections (except for personal collections)
+Metabase only includes some artifacts its exports.
+
+- Collections (except for personal collections, unless specified by the `--user` flag)
 - Dashboards
 - Saved questions
+- Actions
+- Models
 - Segments and Metrics defined in the Data Model
+- SQL Snippets
 - Public sharing settings for questions and dashboards
 - Admin Panel settings, except for permissions
```

Comment:
Maybe we should remove this line. It's not clear whether this refers to "Admin Panel > Settings" sidebar or the Admin Panel top nav (like Databases, Data Model, etc.), or both.

Maybe it is easier to list the Admin Panel settings explicitly?
```
- Some Admin Panel settings, including:
  - Data Model settings (including Segments and Metrics)
  - Database connection settings
  - Public sharing settings
```

This could be annoying if we have to list everything in the "Admin Panel > Settings" sidebar though.

Also, are People settings included?

Reply:
I added the list of settings that's actually exported, which should cover everything.